packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -462,4 +457,16 @@ export const run = async () => {
   if (requiredEnvVars.RUN_STATE_BATCH_SUBMITTER) {
     loop(() => stateBatchSubmitter.submitNextBatch())
   }
+
+  if (
+    config.bool('run-prometheus-server', env.RUN_PROMETHEUS_SERVER === 'true')
```

Comment:
Personally I think that `if` statements that look like:

```
if (
  ...
) {
  ...
}
```

Are ugly and harder to read

Reply:
linter automatically makes it that way because of the length :/

Reply:
Its possible to avoid that by doing a variable assignment 1 line above and it ends up making it take up less lines of code